packages/sonarwhal/.sonarwhalrc

```diff
@@ -27,6 +27,7 @@
         "meta-viewport": "warning",
         "minified-js": "warning",
         "no-bom": "warning",
+        "no-broken-links": "warning",
```

Comment:
@alrra if we modify the `.sonarwhalrc` file then we publish a new version, right? I'm not sure how to deal with this. At least it should be a separate commit (haven't checked if it is the case already though).

Reply:
Changes to `.sonarwhalrc` should be in a different commit.